wallet/src/wallet/mod.rs

```diff
@@ -283,7 +283,7 @@ impl<B: storage::Backend> Wallet<B> {
             .accounts
             .get(&account_index)
             .ok_or(WalletError::NoAccountFoundWithIndex(account_index))?;
-        let utxos = account.get_utxos(utxo_types, self.latest_median_time);
+        let utxos = account.get_utxos(utxo_types, self.latest_median_time, false);
```

Comment:
Should we make this an enum instead of bool, with support for Unconfirmed and/or Locked Utxos?

Reply:
I was thinking the bool could be combined with `utxo_types`, but looking at it I think it's not really a UTXO type but a state, so I would keep it separate. Maybe a UTXO State would be useful, then you could add `Confirmed`, `Unconfirmed`, `Valid`, `Invalid`, `Orphaned` etc.

Reply:
We have a TxState enum that is used, but this is a different thing, it is supposed to be an enum from wallet commands, e.g. for get_balance/ show utxos to show only confirmed balance, or unconfirmed, or confirmed including still locked utxos...